workflow/ignite/handlers/learning_rate/warmup.py

```diff
@@ -0,0 +1,9 @@
+
+
+def warmup(n_steps):
+    def _warmup(step, multiplier):
```

Comment:
I really like this naming approach since there is no good way of returning anonymous functions in python

Reply:
A good reason not to name the wrapped function something generic like "wrapped_fn".